frontend/src/components/Candidate-Decider/CandidateDecider.tsx

```diff
@@ -17,6 +17,14 @@ type CandidateDeciderProps = {
 };
 
 const CandidateDecider: React.FC<CandidateDeciderProps> = ({ uuid }) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [isBypassingModal, setIsBypassingModal] = useState(false);
+  const [navigationDirection, setNavigationDirection] = useState<
+    'next' | 'previous' | 'other' | null
+  >(null);
+  const [navigationDirectionCandidate, setNavigationDirectionCandidate] = useState<number | null>(
```

Comment:
rename? "navigationDirection" doesn't really apply anymore if we're not tracking navigation direction